src/test/java/ru/training/at/hw8/TrelloBoardTests.java

```diff
@@ -0,0 +1,177 @@
+package ru.training.at.hw8;
+
+import static ru.training.at.hw8.constants.ResponseField.*;
+import static ru.training.at.hw8.core.TrelloServiceObject.*;
+import static ru.training.at.hw8.matchers.Matchers.*;
+
+
+import io.restassured.http.Method;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import ru.training.at.hw8.beans.TrelloResponse;
+import ru.training.at.hw8.constants.PermissionLevel;
+import ru.training.at.hw8.core.DataProviderForTrello;
+
+public class TrelloBoardTests {
+    @Test(dataProvider = "createDefaultBoardDataProvider",
```

Comment:
USE ctrl+alt+l - to format the code and delete unused imports. Sometimes the code won't compile because of the imports